Sources/SwiftDocC/Infrastructure/Symbol Graph/GeneratedDocumentationTopics.swift

```diff
@@ -189,7 +189,7 @@ enum GeneratedDocumentationTopics {
         )
         
         let collectionTaskGroups = try AutomaticCuration.topics(for: temporaryCollectionNode, withTrait: nil, context: context)
-            .map({ AutomaticTaskGroupSection(title: $0.title, references: $0.references, renderPositionPreference: .bottom) })
+            .map({ AutomaticTaskGroupSection(title: $0.title!, references: $0.references, renderPositionPreference: .bottom) })
```

Comment:
Are we confident this will never be `nil` here?

Reply:
Yes, the value here comes from `AutomaticCuration.ReferenceGroup.title` which is non-nil.

Reply:
Sounds good. That isn't really clear to me from the context so a comment to that effect might be nice here.